src/views/Movies-Components/SearchBar.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { Grid } from "@mui/material";
+import TextField from "@mui/material/TextField";
+import Autocomplete from "@mui/material/Autocomplete";
+export const SearchBar = () => {
+  const [autoCompleteValue, setAutoCompleteValue] = useState();
+  return (
+    <Grid item pt={1}>
+      <Autocomplete
+          sx={{ width: 260 ,pl:142 }}
+          freeSolo
+          id="free-solo-2-demo"
+          disableClearable
+         renderInput={(params) => (
+          <TextField
+            {...params}
+            placeholder="Search Bar"
+            InputProps={{
+              ...params.InputProps,
+              type: "search",
+            }}
+            value={autoCompleteValue}
+            onChange={(e) => {
+              setAutoCompleteValue(e.target.value);
+            }}
+            sx={{
+                "& .Mui-focused": {
+                    "& .MuiOutlinedInput-notchedOutline": {

```

Comment:
How did you learn this CSS?

Reply:
style overwrite  noman bhai sy sekha tha unho ny weather app keya tha thore bot he ata ha bs mujhy